### PR TITLE
Jtesar/chapter1 general installation

### PR DESCRIPTION
**Pull request by jtesar-rh**

### CODE REVIEW

**Comment by rsriniva** [comment]
@jhurlocker and @strangiato - please review this short section

File: modules/chapter1/pages/section1.adoc [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+= General Information about Installation   
 
-This is _Section 1_ of _Chapter 1_ in the *hello* quick course....
\ No newline at end of file
+*Red{nbsp}Hat Openshift Data Science* is distributed as an operator and available through OpenShift marketplace. There are some other operators that you may need to install depending on which features and components of *Red{nbsp}Hat Openshift Data Science* you want to install and use.
+
+https://www.redhat.com/en/technologies/cloud-computing/openshift/pipelines[Red{nbsp}Hat Openshift Pipelines Operator]::
+ This operator is required for the *Red{nbsp}Hat Openshift Data Science* pipelines component.
+
+https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html[NVIDIA GPU Operator]::
```
Comment (jramcast) [inline modules/chapter1/pages/section1.adoc:9]: Same as in the other 2 elements, add a short description.

```suggestion
https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html[NVIDIA GPU Operator]::
This operator is required for GPU support in  Red Hat Openshift Data Science.
```

File: modules/chapter1/pages/section1.adoc [hunk 2]
```diff
@@ -1,3 +1,14 @@
-= Section 1
+//find a better title
+= General Information about Installation   
 
-This is _Section 1_ of _Chapter 1_ in the *hello* quick course....
\ No newline at end of file
+*Red{nbsp}Hat Openshift Data Science* is distributed as an operator and available through OpenShift marketplace. There are some other operators that you may need to install depending on which features and components of *Red{nbsp}Hat Openshift Data Science* you want to install and use.
```
Comment (strangiato) [inline modules/chapter1/pages/section1.adoc:4]: What about something like this:

*Red{nbsp}Hat Openshift Data Science* is available to install a self-managed version as an operator through OperatorHub or as a fully managed solution through OpenShift Marketplace.

Comment (strangiato) [inline modules/chapter1/pages/section1.adoc:4]: I would recommend breaking the discussion of the pre-reqs out into it's own paragraph.

File: modules/chapter1/pages/section1.adoc [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+Red{nbsp}Hat Openshift Data Science is available to install a self-managed version as an operator through OperatorHub or as a fully managed solution through OpenShift Marketplace. 
+
+There are some other operators that you may need to install depending on which features and components of the *Red{nbsp}Hat Openshift Data Science* you want to install and use.
+
+https://www.redhat.com/en/technologies/cloud-computing/openshift/pipelines[Red{nbsp}Hat Openshift Pipelines Operator]::
+ This operator is required if you want to install the  *Red{nbsp}Hat Openshift Data Science Pipelines* component.
+
+https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html[NVIDIA GPU Operator]::
+https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-node-feature-discovery-operator.html[Node Feature Discovery Operator]::
+This *NVIDIA GPU Operator* is required for GPU support in  Red Hat Openshift Data Science.
```
Comment (jramcast) [inline modules/chapter1/pages/section1.adoc:13]: What about moving this sentence to the NVIDIA GPU Operator item?

Now, the definition list looks a bit strange without any text under the GPU operator term.

image

```suggestion
https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html[NVIDIA GPU Operator]::
This *NVIDIA GPU Operator* is required for GPU support in  Red Hat Openshift Data Science.

https://docs.openshift.com/container-platform/4.13/hardware_enablement/psap-node-feature-discovery-operator.html[Node Feature Discovery Operator]::
```

**Review by jramcast — APPROVED** [review]
I just left a non-blocking comment. Please go ahead and merge.

NOTE:  I also noticed that we are not using the new `Red Hat Openshift.AI` product name but IMO we can do that later, and run `sed` over all courses or something like that. At this point, I am bit confused by the naming, so I would wait until the terminology is a bit more stable.

**Pull request merged**